lesson10/index.html

```diff
@@ -0,0 +1,268 @@
+            src="assets/img/icon-second-tests.svg"
+            alt="Five second tests"
+            title="Five second tests"
+          />
+          <h3 class="fs-400 fw-600">Five second tests</h3>
+          <p>Personal choices and the overall personality of the person.</p>
+        </div>
+      </div>
+      <div>
+        <a href="#" class="link--main">SIGN UP NOW</a>
+      </div>
+    </section>
+
+    <!----------- Design Professionals section ----------->
+
+    <section class="professionals container flex">
+      <div class="professionals__text">
+        <p>Effortless Validation for</p>
+        <h2 class="fs-500 fw-600">Design Professionals</h2>
+        <p>
+          The Myspace page defines the individual, his or her characteristics,
+          traits, personal choices and the overall personality of the person.
+        </p>
+        <p>
+          <span>Accessory makers</span>While most people enjoy casino gambling,
+          sports betting, lottery and bingo playing for the fun
+        </p>
+        <p>
+          <span>Alterationists</span>If you are looking for a new way to promote
+          your business that won’t cost you more money,
+        </p>
+        <p>
+          <span>Custom Design designers</span>If you are looking for a new way
+          to promote your business that won’t cost you more money,
+        </p>
+      </div>
+      <div class="professionals__img">
+        <img
+          src="assets/img/img-design-professionals.png"
+          alt="Design Professionals"
+          title="Design Professionals"
+        />
+      </div>
+    </section>
+
+    <!----------- Product Managers section ----------->
+
+    <section class="product container flex">
+      <div class="product__img">
+        <img
+          src="assets/img/img-product-managers.png"
+          alt="Product Managers"
+          title="Product Managers"
+        />
+      </div>
+      <div class="product__text">
+        <p>Easier decision making for</p>
+        <h2 class="fs-500 fw-600">Product Managers</h2>
+        <p>
+          The Myspace page defines the individual, his or her characteristics,
+          traits, personal choices and the overall personality of the person.
+        </p>
+        <div class="notes flex">
+          <img src="assets/img/icon-correct.svg" alt="" />
+          <p>Never worry about overpaying for your energy again.</p>
+        </div>
+        <div class="notes flex">
+          <img src="assets/img/icon-correct.svg" alt="" />
+          <p>
+            We will only switch you to energy companies that we trust and will
+            treat you right
+          </p>
+        </div>
+        <div class="notes flex">
+          <img src="assets/img/icon-correct.svg" alt="" />
+          <p>We track the markets daily and know where the savings are.</p>
+        </div>
+      </div>
+    </section>
+
+    <!----------- Marketers section ----------->
+
+    <section class="marketers container flex">
```

Comment:
It's ok for now. But in the future, if you will need to replace this BEM block with another place you will need to move it with 'flex' class. Just remember about it )